pkgs/by-name/ut/ut1999/package.nix

```diff
@@ -0,0 +1,122 @@
+{ lib, stdenv, requireFile, autoPatchelfHook, fetchurl,
+libgcc, wxGTK32, innoextract, makeWrapper,
+libGL, SDL2, openal, libmpg123, libxmp }:
+
+let
+  oldUnrealPatch = stdenv.mkDerivation {
+    name = "ut1999-oldunreal-patch";
+    version = "469d";
+    sourceRoot = ".";
+    src = fetchurl {
+      url = "https://github.com/OldUnreal/UnrealTournamentPatches/releases/download/v469d/OldUnreal-UTPatch469d-Linux-amd64.tar.bz2";
+      hash = "sha256-aoGzWuakwN/OL4+xUq8WEpd2c1rrNN/DkffI2vDVGjs=";
+    };
+
+    buildInputs = [
+      libgcc
+      stdenv.cc.cc
+      wxGTK32
+    ];
+
+    nativeBuildInputs = [
+      autoPatchelfHook
+      makeWrapper
+    ];
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out
+      ls -alh
+      cp -r ./* $out
+
+      runHook postInstall
+    '';
+
+    dontBuild = true;
+  };
+
+  unpackGog = stdenv.mkDerivation {
+    name = "ut1999-gog";
+    version = "2.0.0.5";
+
+    src = requireFile rec {
+      name = "setup_ut_goty_2.0.0.5.exe";  # FIXME: use official name
```

Comment:
What does this mean?

Reply:
@Atemu , initially I had a hardcoded name that was not the same as the vanilla one from GOG. I've fixed it before but forgot to remove the `FIXME`.